util/util.go

```diff
@@ -895,3 +895,28 @@ func GetVolumeMeta(path string) (*VolumeMeta, error) {
 	}
 	return meta, nil
 }
+
+func TrimFilesystem(name string) error {
+	nsPath := iscsi_util.GetHostNamespacePath(HostProcPath)
+	nsExec, err := iscsi_util.NewNamespaceExecutor(nsPath)
+	if err != nil {
+		return err
+	}
+
+	mountOutput, err := nsExec.Execute("bash", []string{"-c", fmt.Sprintf("mount | grep /dev/longhorn/%s | awk '{print $3}'", name)})
+	if err != nil {
+		return fmt.Errorf("cannot find volume %v mount info on host: %v", name, err)
+	}
+
+	mountList := strings.Split(strings.TrimSpace(mountOutput), "\n")
+	if len(mountList) != 1 {
+		return fmt.Errorf("zero or multiple mount points for volume %s: %+v", name, mountList)
+	}
+
+	_, err = nsExec.Execute("fstrim", mountList)
```

Comment:
Just curious. `fstrim` is a blocking call or a non-blocking call?

Reply:
I think it's a blocking call. I need to have a test (by deleting tons of small files) for this part.

Reply:
I tried to create 50000 4MB files (200G in total) in a 500G volume then delete and trim them, `fstrim` gets stuck for around 2 seconds. I don't think there is a timeout risk.